pkg/plugins/golang/v4/scaffolds/internal/templates/controllers/controller_suitetest.go

```diff
@@ -34,6 +34,10 @@ type SuiteTest struct {
 	machinery.BoilerplateMixin
 	machinery.ResourceMixin
 
+	// K8SVersion define the k8s version used to do the scaffold
+	// so that is possible retrieve the binaries
+	K8SVersion string
+
```

Comment:
@camilamacedo86 I like this change. But have this question on adding this option. Having k8s version here would not necessarily mean that my project is compatible with that particular version of K8s. It would just mean that we are scaffolding our test suite such that binaries for that version are downloaded. Do we need docs on this, or can we rename this to be `K8sTestVersion`?

Reply:
The binaries downloaded are within the version defined on the Makefile. 
Please, feel free to provide any wording suggestion that you see that would fit better and make things more understandable that I will accepted :-)

Reply:
I updated to have the same name used in the Makefile. (ENVTEST_K8S_VERSION)